redash/models.py

```diff
@@ -277,33 +172,27 @@ def has_access(self, obj, access_type):
         return False
 
 
-class Organization(ModelTimestampsMixin, BaseModel):
+class Organization(TimestampMixin, db.Model):
     SETTING_GOOGLE_APPS_DOMAINS = 'google_apps_domains'
     SETTING_IS_PUBLIC = "is_public"
 
-    id = peewee.PrimaryKeyField()
-    name = peewee.CharField()
-    slug = peewee.CharField(unique=True)
-    settings = JSONField()
+    id = Column(db.Integer, primary_key=True)
+    name = Column(db.String(255))
+    slug = Column(db.String(255), unique=True)
+    settings = Column(PseudoJSON)
 
-    class Meta:
-        db_table = 'organizations'
+    __tablename__ = 'organizations'
 
     def __repr__(self):
         return u"<Organization: {}, {}>".format(self.id, self.name)
 
-    # When Organization is used with LocalProxy (like the current_org helper), peewee doesn't recognize it as a Model
-    # and might call int() on it. This method makes sure it works.
-    def __int__(self):
-        return self.id
-
     @classmethod
     def get_by_slug(cls, slug):
-        return cls.get(cls.slug == slug)
+        return cls.query.filter(cls.slug == slug).first()
```

Comment:
I suspect this should also be a `one()` or `one_or_none()` as I think you want errors if more than one result is returned for a given slug.